articles/introduction-to-c#/index .md

```diff
@@ -0,0 +1,212 @@
+### Introduction to C#
+C# is an Object-Oriented programming language and it is portable. C# was invented in 2002 and it is one of the programming languages widely used today.
+
+C# is pronounced as "C-Sharp".
+
+Programs written in C# utilize the [.NET Framework](https://dotnet.microsoft.com/download/dotnet-framework/net45) to run. .NET is a framework used to make applications and run the .NET software. It was created by Microsoft and it runs primarily on Windows operating system. There is an open-source implementation called [Mono](https://www.mono-project.com/download/stable/) which conveys a cross-platform.
```

Comment:
`There is an open-source implementation called Mono which conveys a cross-platform.`

What do you mean by which conveys a cross-platform? Can you rewrite it to be more concise?